src/test/scala/com/cognite/sdk/scala/common/DataModelPropertiesSerializerTest.scala

```diff
@@ -57,7 +57,9 @@ class DataModelPropertiesSerializerTest extends AnyWordSpec with Matchers {
     res.isLeft shouldBe true
     val Left(decodingFailure) = res
     val error = DecodingFailure.unapply(decodingFailure)
-    error.map(_._1).getOrElse("").contains(propType) shouldBe true
+    val errorMessage =  error.map(_._1).getOrElse("")
+    errorMessage.contains("with wrong type, expecting") || errorMessage.contains("could not be parsed") ||
+      errorMessage.contains(propType) shouldBe true
```

Comment:
This is because circe had some [changes to improve error messages,](https://github.com/circe/circe/pull/1617) so it's not the same like before